services/meta/association.md

```diff
@@ -1,45 +1,75 @@
+out  | evt.association.report     | object     | `max_supports` | Reports members of a specific association. Property `max_supports` defines how many devices can be added to this group. See [`association_members`](#definitions) for value definition.  |
+in   | cmd.groups_info.get_report | null       |                | Z-Wave only. Request detailed information about supported association groups.                                                                                                            |
+out  | evt.groups_info.report     | object     |                | Z-Wave only. Returns a list of supported associations groups with descriptions. See [`association_info_object`](#definitions).                                                           |
+
+## Service Props
+
+Name             | Supported Values                                | Description                                                             |
+-----------------|-------------------------------------------------|-------------------------------------------------------------------------|
+`in_services`    | `["out_bin_switch", "out_lvl_switch", "color"]` | Zigbee only. List of services that can be controlled, e.g. on a lamp.   |
+`out_services`   | `["out_bin_switch", "out_lvl_switch", "color"]` | Zigbee only. List of services the device can control, e.g. on a button. |
+
+> A controllable device (like a lamp) would have a list of `in_services` that would indicate the services it supports. A controller (like a button) would have a list of `out_services` instead.
+
+## Definitions
+* `association_members` is an object describing association group members:
+ 
+| Field        | Type      | Example           | Description                                    |
+|--------------|-----------|-------------------|------------------------------------------------|
+| `group`      | int       | `1, 3, 5`         | Id of the supported group.                     |
+| `members`    | str_array | `["2_0", "15_1"]` | List of group members.                         |
+
+> The `group` field is completely ignored for zigbee. The field can be null, empty or removed completely for commands and will be missing for events. Zigbee-ad internally creates a group with an ID that's equal to the device ID.
+
+* `association_info` is an object describing a specific association group:
+
+| Field        | Type   | Example                                   | Description                                                       |
+|--------------|--------|-------------------------------------------|-------------------------------------------------------------------|
+| `group_num`  | int    | `1, 3, 5`                                 | Id of the supported group.                                        |
```

Comment:
the same as in the case of association_members.group

Reply:
Also maybe the field should be just `group` if in other objects the same thing is represented by `group` property.